velox/expression/SimpleFunctionAdapter.h

```diff
@@ -69,9 +70,9 @@ class SimpleFunctionAdapter : public VectorFunction {
 
   template <int32_t POSITION>
   static constexpr bool isArgFlatConstantFastPathEligible =
```

Comment:
It would be nice to add a comment explaining what makes the function call eligible for this fast path.

Reply:
> eqToConstant

I will check if those are consistent, i will test them locally.